resources/qml/Dialogs/AboutDialog.qml

```diff
@@ -95,23 +92,45 @@ UM.Dialog
         UM.Label
         {
             text: description
-            visible: text !== ""
             Layout.fillWidth: true
             Layout.preferredWidth: 2
         }
 
         UM.Label
         {
-            text: license
-            visible: text !== ""
+            text:
+            {
+                if (license_full !== "")
+                {
+                    return `<a href="license_full">${license}</a>`;
+                }
+                else
+                {
+                    return license;
+                }
+            }
             Layout.fillWidth: true
             Layout.preferredWidth: 1
+
+            Component
+            {
+                id: componentLicenseDialog
+
+                LicenseDialog
+                {
+                }
```

Comment:
Did you mean to add anything in here? -- I'm used to those being on the same line if not 😅

Reply:
Yes indeed, missed that :smile: